### PR TITLE
Admin settings placeholder page

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/stylesheets/admin/page-settings.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    width: 30em;
+
+    @include screen-xs-max {
+      display: inline;
+      margin-right: 10px;
+    }
+  }
+
+  .btn-add-schedule {
+      margin: 20px 0;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-settings.scss]: Line should be indented 4 spaces, but was indented 6 spaces

File: app/policies/setting_policy.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class SettingPolicy < ApplicationPolicy
+  def index?
+    true
```
Comment (RuslanHamidullin) [inline app/policies/setting_policy.rb]: @hammerfunk @jaleszek I suppose that here should be:
```
admin.admin?
```

**Comment by RuslanHamidullin** [comment]
All looks fine :+1: 
Just need to correct SettingPolicy in app/policies/setting_policy.rb
should be:
```
class SettingPolicy < ApplicationPolicy
  def index?
    admin.admin?
  end
end
```

File: app/policies/setting_policy.rb (hunk 2 above)
Comment (jaleszek) [inline app/policies/setting_policy.rb]: yeah, I will change when (if) assessor extraction pr will be merged

Comment (RuslanHamidullin) [inline app/policies/setting_policy.rb]: thanks @jaleszek :beer:

**Pull request merged**